### DISCUSSION TITLE
IPv6无法代理？

### DISCUSSION THREAD

**Discussion by UchihaYuki** [vernesong/OpenClash#1985] [Q&A]
看到2020年的一个帖子说，IPv6都是做直连的，现在还是吗？
clash的配置文件里面可以配置ipv6:true, 那个只是说可以解析域名到ipv6地址，但是连接的时候只能走直连模式？

**Comment by vernesong** [comment]
代理和DNS是两个开关控制，开启后确保节点支持IPV6不然会连接失败，下面设备的网关和dns也要改
